templates/common/baremetal/files/NetworkManager-resolv-prepender.yaml

```diff
@@ -49,6 +47,39 @@ contents:
             show \
             "{{.Infra.Status.PlatformStatus.BareMetal.APIServerInternalIP}}" \
             "{{.Infra.Status.PlatformStatus.BareMetal.IngressIP}}")
+        # check if we got DHCPv6 hostname. The value would usually be in
+        # DHCP6_FQDN_FQDN but NM internal client does not parse the option
+        # NM only parses it for IPv4
+        if [[ "$NAMESERVER_IP" =~ ":" ]]; then
+            LEASE_FILE=$(mktemp)
+            const CONT_NAME="dhcpv6-hostname-workaround"
```

Comment:
const is not a thing in bash. I think you want readonly instead.

Reply:
Damn it! This happens to me for helping somebody fix some javascript earlier this week.